react-native-gutenberg-bridge/index.js

```diff
@@ -16,6 +16,10 @@ export const mediaSources = {
 	siteMediaLibrary: 'SITE_MEDIA_LIBRARY',
 };
 
+export const userEvents = {
+	pageTemplateApplied: 'page_template_applied',
```

Comment:
I was going to comment that this might look better in uppercase, but seeing the `mediaSources` above it's probably best to leave it as is for consistency.